velox/exec/OperatorUtils.h

```diff
@@ -75,6 +75,15 @@ VectorPtr wrapChild(
 RowVectorPtr
 wrap(vector_size_t size, BufferPtr mapping, const RowVectorPtr& vector);
 
+// Wraps all children of the specified row vector into a dictionary using
```

Comment:
> specified row vector

This comment suggests that the function takes a RowVector, but it takes rowType + childVectors instead. Would you update the comment or change the signature?

> Returns an empty vector if mapping is null.

There is another wrap function just above this one with very similar interface, but different semantic: "Returns vector as-is if mapping is null." Wouldn't it be confusing to not know for sure whether wrap returns an empty vector or input vector as-is when mapping is null?